src/Http/Http.Features/src/IFeatureCollection.cs

```diff
@@ -33,13 +33,13 @@ public interface IFeatureCollection : IEnumerable<KeyValuePair<Type, object>>
         /// </summary>
         /// <typeparam name="TFeature">The feature key.</typeparam>
         /// <returns>The requested feature, or null if it is not present.</returns>
-        TFeature Get<TFeature>();
+        TFeature? Get<TFeature>();
```

Comment:
Do you think we need a `GetRequiredFeature` to avoid the use of null forgiveness?

Reply:
No. It's not that commonly used API. But it could be added later.

Reply:
Yes, GetRequiredFeature would be a good idea. There are several features we always assume are present like IHttpRequestFeature.